index.html

```diff
@@ -1,28 +1,231 @@
+            <h3 class="mb-0">Projects</h3>
+          </div>
+          <p class="mb-0">
+            Completing hands-on projects to apply the skills learned throughout
+            the course and build a portfolio of web development projects.
+          </p>
+        </div>
+      </div>
+      <br />
+      <button class="enrol">Enroll now</button>
+      <h5>
+        And take the first step towards a rewarding career in web development!"
+      </h5>
+    </section>
 
+    <section class="speak" id="speaker">
+      <h2>Feature Professors</h2>
+      <hr class="w-50 text-light" />
+      <div class="speaker d-flex flex-column flex-md-row" id="prof"></div>
+      <button
+        class="d-md-none w-50 d-flex justify-content-center align-items-center"
+      >
+        <h3 class="mb-0">More</h3>
+        <i class="fa-solid fa-chevron-down text-danger"></i>
+      </button>
+    </section>
 
+    <section class="about" id="about">
+      <h2 class="text-center">
+        JavaScript BootCamp for scratch to professional
+      </h2>
+      <p class="bg-light detail border-2 p-3 m-0 text-center">
+        Welcome to our JavaScript Bootcamp! Our course is designed to teach you
+        the fundamentals of JavaScript programming from the ground up. ced
+        instructors will guide you through hands-on exercises and real-world
+        projects to help you build a solid foundation in JavaScript programming.
+        You'll also have access to our online community, where you can ask
+        questions and get feedback from other students and instructors.
+      </p>
+      <h2>Best JavaScript Course</h2>
+      <hr />
+      <p class="text-center w-50">
+        We are excited to kick off our event and share our passion for coding
+        with you. We have designed this event to cater to both beginners and
+        experienced developers who want to expand their skill set.
+      </p>
+      <div
+        class="d-flex align-items-center justify-content-center p-3 event border border-2"
+      >
+        <i class="fa-brands fa-square-js m-2 js"></i>
+        <h3>
+          JavaScript <br />
+          2023
+        </h3>
+      </div>
+      <hr />
+      <h3>See next sessions</h3>
+      <hr />
+      <h5>Take a look to our next BootCamp on React and Ruby</h5>
+      <div class="d-flex flex-column flex-md-row mb-3 even">
+        <div class="next text-center">
+          <h2>Ruby</h2>
+          <br />
+          <h4>The full Ruby course on 17 May</h4>
+        </div>
+        <div class="next text-center">
+          <h2>React</h2>
+          <br />
+          <h4>The full React course on 17 July</h4>
+        </div>
+      </div>
+    </section>
+    <section class="partner">
+      <h2 class="text-center">Our partner</h2>
+      <hr />
+      <div class="d-flex flex-wrap gap-2 part-name">
+        <h3>Mozilla</h3>
+        <h3>Google</h3>
+        <h3>Aple</h3>
+        <h3>Uber</h3>
+        <h3>Amazone</h3>
+      </div>
+    </section>
```

Comment:
- [ ] This is content of about page which is supposed to be a seperate file. Kindly create a seperate file and put all these content in the file and link the file to your index.html file.

### Project Requirements
![number-of-pages](https://user-images.githubusercontent.com/26597432/225553714-e6fd31ad-d343-48ef-b1d6-761b96e95ec1.png)